Jenkinsfile

```diff
@@ -180,30 +184,25 @@ pipeline {
         LOG_SERVER = "logs.opencontrail.org"
         LOG_SERVER_USER = "zuul-win"
         LOG_ROOT_DIR = "/var/www/logs/winci"
-        BUILD_SPECIFIC_DIR = "${ZUUL_UUID}"
-        JOB_SUBPATH = env.JOB_NAME.replaceAll("/", "/job/")
-        REMOTE_DST_FILE = "${LOG_ROOT_DIR}/${BUILD_SPECIFIC_DIR}/log.txt.gz"
     }
 
     post {
         always {
             node('master') {
                 script {
-                    // Job triggered by Zuul -> upload log file to public server.
-                    // Job triggered by Github CI repository (variable "ghprbPullId" exists) -> keep log "private".
-
-                    // TODO JUNIPER_WINDOWSSTUBS variable check is temporary and should be removed once
-                    // repository contrail-windows is accessible from Gerrit and it is main source of
-                    // windowsstubs code.
-                    if (env.ghprbPullId == null && env.JUNIPER_WINDOWSSTUBS == null) {
-                        // unstash "buildLogs"
-                        // TODO correct flags for rsync
-                        sh "ssh ${LOG_SERVER_USER}@${LOG_SERVER} \"mkdir -p ${LOG_ROOT_DIR}/${BUILD_SPECIFIC_DIR}\""
-                        def tmpLogFilename = "clean_log.txt.gz"
-                        obtainLogFile(env.JOB_NAME, env.BUILD_ID, tmpLogFilename)
-                        sh "rsync $tmpLogFilename ${LOG_SERVER_USER}@${LOG_SERVER}:${REMOTE_DST_FILE}"
-                        deleteDir()
-                    }
+                    def logServer = [
+                        addr: env.LOG_SERVER,
+                        user: env.LOG_SERVER_USER,
+                        rootDir: env.LOG_ROOT_DIR
+                    ]
+                    def destDir = decideLogsDestination(logServer, env.ZUUL_UUID)
+
+                    unstash 'testReport'
```

Comment:
Complementary stash may never happen.
Especially when DONT_CREATE_TESTBEDS is set.
It may likely affect successful execution of a pipeline.